packages/framework/src/Model/Product/Product.php

```diff
@@ -160,19 +160,6 @@ class Product extends AbstractTranslatableEntity
      */
     protected $outOfStockAvailability;
 
-    /**
-     * @var \Shopsys\FrameworkBundle\Model\Product\Availability\Availability
-     * @ORM\ManyToOne(targetEntity="Shopsys\FrameworkBundle\Model\Product\Availability\Availability")
-     * @ORM\JoinColumn(name="calculated_availability_id", referencedColumnName="id", nullable=false)
-     */
-    protected $calculatedAvailability;
```

Comment:
conditionally removed column (it does not exist on previous CC project, but it does on FW based projects) – 99c6c09bbf